doc/rst/language/spec/interoperability.rst

```diff
@@ -337,6 +356,30 @@ in Chapel you would refer to this ``struct`` via
        var x, y, z: real(64);
      }
 
+Note that the above examples apply for C unions as well, so an example
+for non-typedef'd C ``union`` would be like this:
+
+.. code-block:: chapel
+
+      union _noTypedefUnion {
+         float x;
+         double y;
+         int64_t z;
+      };
+
+referring to this ``union`` would be allowed in Chapel, via:
+
+
+
+.. code-block:: chapel
+
+     extern "union _noTypedefUnion" record _noTypedefUnion {
+         var x: real(32);
+         var y: real(64);
+         var z: int(64);
+     }
```

Comment:
Here too, I'd remove all of the underscores, as I don't think they add value and don't match the struct case.

Reply:
Sure!